pandas/tests/arrays/test_integer.py

```diff
@@ -218,25 +218,68 @@ def test_arith_integer_array(self, data, all_arithmetic_operators):
     def test_arith_series_with_scalar(self, data, all_arithmetic_operators):
         # scalar
         op = all_arithmetic_operators
+        if op == '__rpow__':
+            raise pytest.skip("__rpow__ tested separately.")
 
         s = pd.Series(data)
         self._check_op(s, op, 1, exc=TypeError)
 
+    def test_arith_series_with_scalar_rpow(self, data):
+        s = pd.Series(data)
+        # 1^x is 1.0 for all x, so test separately
+        result = 1 ** s
+        expected = pd.Series(1, index=s.index, dtype=data.dtype.numpy_dtype)
```

Comment:
this is adding a lot of duplicated code to test, can you use _check_op here for the 1 case

Reply:
I started down that route, but couldn't make it work. I found it hard to follow all the indirection.

I'm ok with some duplicated code in these tests, to make it clearer what's actually being tested.